### DISCUSSION TITLE
How can people from India donate for this app

### DISCUSSION THREAD

**Discussion by dipayand** [termux/termux-packages#8748] [Q&A] [answered]
How can we donate for Termux

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-packages/wiki/Donate

**Reply by dipayand** [reply]
This is not in Indian currency. When Termux was available in Google, we could pay for Termux:Styling or Termux:Widget. Indian payment system should be possibie. Payment in INR. Bitcoin is not so popular in India

**Reply by agnostic-apollo** [reply]
Apps are currently still available on google play store but they are deprecated and not to be used. You can buy them and uninstall.

You can also subscribe on patreon and liberapay, it should work with your local debit/credit card and currency conversion will automatically be done. Even play store would convert currency internally to reach fornwall.

**Reply by dipayand** [reply]
I already paid for them when it was available on Google Play. Then I moved to F-Droid. Thanks anyway

**Comment by kcubeterm** [comment]
I could help here. I could make an UPI id for Indian user as I live in India as well( but a citizen of mars)  and then will transfer those money in all dev address.  But question here is, could Indian trust a martian guy ?
